[2023-05-26 06:08:35,255] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-03-14T00:00:00+00:00 [queued]>
[2023-05-26 06:08:35,269] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-03-14T00:00:00+00:00 [queued]>
[2023-05-26 06:08:35,270] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 06:08:35,271] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-05-26 06:08:35,272] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 06:08:35,287] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): db_connect> on 2023-03-14 00:00:00+00:00
[2023-05-26 06:08:35,293] {standard_task_runner.py:52} INFO - Started process 973 to run task
[2023-05-26 06:08:35,296] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tercera_entrega_ecentoz', 'db_connect', 'scheduled__2023-03-14T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/dag_tercera_entrega.py', '--cfg-path', '/tmp/tmpy0juonvf', '--error-file', '/tmp/tmp1_dpxlmh']
[2023-05-26 06:08:35,298] {standard_task_runner.py:80} INFO - Job 102: Subtask db_connect
[2023-05-26 06:08:35,367] {task_command.py:371} INFO - Running <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-03-14T00:00:00+00:00 [running]> on host f4aa0d8de4f3
[2023-05-26 06:08:35,437] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ecentoz
AIRFLOW_CTX_DAG_ID=tercera_entrega_ecentoz
AIRFLOW_CTX_TASK_ID=db_connect
AIRFLOW_CTX_EXECUTION_DATE=2023-03-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-14T00:00:00+00:00
[2023-05-26 06:08:58,257] {python.py:173} INFO - Done. Returned value was: None
[2023-05-26 06:08:58,271] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tercera_entrega_ecentoz, task_id=db_connect, execution_date=20230314T000000, start_date=20230526T060835, end_date=20230526T060858
[2023-05-26 06:08:58,320] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-26 06:08:58,344] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
